client/my-sites/site-settings/style.scss

```diff
@@ -142,6 +142,10 @@
 		float: left;
 		line-height: 40px;
 	}
+
+	fieldset.site-icon-setting {
```

Comment:
>Should we apply this in `site-icon-setting/style.scss` instead?

I don't think so, because the styles here aren't relevant to the component itself, but rather its placement amongst other fieldsets on the settings form.